nevergrad/optimization/optimizerlib.py

```diff
@@ -9,6 +9,8 @@
 from collections import deque
 import warnings
 import cma
+from modcma import AskTellCMAES
```

Comment:
Let's avoid importing here otherwise we will need to have a strong dependency to modcma, let's import inline instead

Reply:
Moved.